### ISSUE TITLE
Extend ruleset with MITRE attack technique ID data

### ISSUE THREAD

**Issue by danimegar** [wazuh/wazuh-ruleset#500]
|Epic issue|Branch|
|---|---|
|[#3709](https://github.com/wazuh/wazuh/issues/3709)|[3.10-mitre](https://github.com/wazuh/wazuh-ruleset/compare/3.10...3.10-mitre)|

### Goal

I want ruleset to include references to the attack technique. For that, I will add Mitre technique IDs to some rules. Mitre technique have to match the rule. A sample could be:

```
<rule id="5712" level="10" frequency="8" timeframe="120" ignore="60">
    <if_matched_sid>5710</if_matched_sid>
    <description>sshd: brute force trying to get access to </description>
    <description>the system.</description>
    <mitre>
      <id>T1110</id>
    </mitre>
    <same_source_ip />
    <group>authentication_failures,pci_dss_11.4,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_SI.4,nist_800_53_AU.14,nist_800_53_AC.7,</group>
  </rule> 
```
This rule is related to ssh brute force as well as  Mitre Technique [T1110](https://attack.mitre.org/techniques/T1110/). All Mitre techniques can be consulted in this [link](https://attack.mitre.org/).

**Comment by danimegar** [comment]
I added Mitre Techniques to some rules, mostly related to ssh brute force. I added a technique concerning Sudo to another rule, too.

**Comment by danimegar** [comment]
I merged branch 3.11 to 3.10-mitre in order to update the branch

**Comment by danimegar** [comment]
Currently, MITRE information has been added to 490 rules, the majority being between level 7 and 15.
